app/internal/notify/client.go

```diff
@@ -77,54 +84,90 @@ type errorItem struct {
 func (c *Client) TemplateID(name string) string {
 	if c.isProduction {
 		switch name {
-		case "MLPA Beta signature code":
+		case "MLPA Beta signature code - Email":
```

Comment:
I should probably have exported this as a const, but now there are 2 it would make even more sense. Then it could be simplified to an int using `iota` since we don't care about the string value